### DISCUSSION TITLE
Making sure I have prerequisites  for validator tool

### DISCUSSION THREAD

**Discussion by Orange-1234** [CMSgov/price-transparency-guide-validator#42] [Q&A]
I have not used Node, NPM, Git or Docker before.  I believe I have downloaded and installed the first 3.  Where do I find the download for Docker?  When I do a search for  Docker 19 all I get is "Dockers Desktop" and it is version 4.6 - is that what I am supposed to be downloading?  Also during the webinar when Scott H did a demo of the validator tool I am unclear on what he was in when he ran the git command,, was checking to see if docker was running, etc.  Was that a command line window?

**Comment by jlamb0423** [comment]
The found the instructions to be pretty straight forward - but you do have to install node.js, git and docker from their respective web sites.

**Reply by Rashmi-sb** [reply]
we did not see issue with installation , we see the tool is not validating the IP json correctly .

**Comment by mint-thompson** [comment]
Thank you for your questions! The version number given for Docker refers to the version of the engine. The 4.6 version you found is likely for the desktop client, which has different version numbers. And you are correct about what you saw in the webinar: Scott ran the git and docker commands from a command line window.

**Reply by Orange-1234** [reply]
Thank you for your answers!  I installed the Docker for Desktop 4.6 as I wasn't able to figure how to install only the Docker engine.  I am able to use the Docker desktop client version to run the validator tool.
